### PR TITLE
[hi] Updated documentation of  _index.md (overview page)

### PR DESCRIPTION
**Pull request by AnshRoshan**
changed the overview pages.

### CODE REVIEW

**Comment by AnshRoshan** [comment]
@divya-mohan0209 thanks for the advice. I sync the fork and update. But discarding the commit closed the PR. So, I created this PR.

File: content/hi/docs/concepts/overview/_index.md [hunk 1]
```diff
@@ -21,70 +21,70 @@ sitemap:
 कुबेरनेट्स नाम ग्रीक से उत्पन्न हुआ है, जिसका अर्थ है हेल्समैन या पायलट।
 K8s एक संक्षिप्त नाम के रूप में "K" और "s" के बीच आठ अक्षरों को गिनने का परिणाम है। Google ने 2014 में कुबेरनेट्स प्रोजेक्ट को ओपन-सोर्स किया। कुबेरनेट्स [गूगल के 15 से अधिक वर्षों के अनुभव](/blog/2015/04/borg-predecessor-to-kubernetes/) को समुदाय से सर्वोत्तम नस्ल के विचारों और प्रथाओं के साथ बड़े पैमाने पर उत्पादन कार्यभार को जोड़ती है।
 
-## काल में वापस जाना
+## समय में वापस जाना
```
Comment (kundan2707) [inline content/hi/docs/concepts/overview/_index.md:24]:
```suggestion
## समय में पीछे जाना
```

File: content/hi/docs/concepts/overview/_index.md [hunk 2]
```diff
@@ -21,70 +21,70 @@ sitemap:
...
 
-आइए काल में वापस जाकर एक नज़र डालते हैं कि कुबेरनेट्स इतना उपयोगी क्यों है।
+आइए वक्त में पीछे जाकर एक नज़र डालते हैं कि कुबेरनेट्स इतना उपयोगी क्यों है।
```
Comment (kundan2707) [inline content/hi/docs/concepts/overview/_index.md:26]: follow same convention everywhere like either समय or वक्त similiarly either वापस or  पीछे

File: content/hi/docs/concepts/overview/_index.md [hunk 3]
```diff
@@ -21,70 +21,70 @@ sitemap:
...
+**पारंपरिक डिप्लॉयमेंट दौर:**
+प्रारंभ में, संगठनों ने भौतिक (physical) सर्वरों पर एप्लिकेशन चलाए। भौतिक सर्वर में एप्लिकेशनों के लिए संसाधन सीमाओं को परिभाषित करने का कोई तरीका नहीं था, और इससे संसाधनों के आवंटन की समस्याएँ उत्पन्न हुईं। उदाहरण के लिए, यदि एक से अधिक एप्लिकेशन एक ही भौतिक सर्वर पर चलते हैं, तो ऐसी परिस्थिति हो सकती है, जहाँ एक एप्लिकेशन अधिकांश संसाधनों को ले लेगा, और इसके परिणामस्वरूप, अन्य एप्लिकेशनों का प्रदर्शन खराब हो सकता है। इसका एक समाधान यह होगा कि प्रत्येक एप्लिकेशन को एक अलग भौतिक सर्वर पर चलाया जाए। लेकिन यह पैमाना नहीं था क्योंकि संसाधनों का कम उपयोग किया गया था, और संगठनों के लिए कई भौतिक सर्वरों को बनाए रखना महंगा था।
```
Comment (kundan2707) [inline content/hi/docs/concepts/overview/_index.md:31]:
```suggestion
प्रारंभ में, संगठनों ने भौतिक (physical) सर्वरों पर एप्लिकेशन चलाए। भौतिक सर्वर में एप्लिकेशनों के लिए संसाधन सीमाओं को परिभाषित करने का कोई तरीका नहीं था, और इससे संसाधन आवंटन समस्याएं उत्पन्न हुईं। उदाहरण के लिए, यदि एक से अधिक एप्लिकेशन एक भौतिक सर्वर पर चलते हैं, तो ऐसे उदाहरण हो सकते हैं, जहाँ एक एप्लिकेशन अधिकांश संसाधनों को ले लेगा, और इसके परिणामस्वरूप, अन्य एप्लिकेशनों का प्रदर्शन खराब हो सकता है। इसका एक समाधान यह होगा कि प्रत्येक एप्लिकेशन को एक अलग भौतिक सर्वर पर चलाया जाए। लेकिन यह पैमाना नहीं था क्योंकि संसाधनों का कम उपयोग किया गया था, और संगठनों के लिए कई भौतिक सर्वरों को बनाए रखना महंगा था।
```

File: content/hi/docs/concepts/overview/_index.md [hunk 4]
```diff
@@ -21,70 +21,70 @@ sitemap:
...
+**वर्चुअलाइज्ड डिप्लॉयमेंट दौर:**  एक समाधान के रूप में, वर्चुअलाइजेशन पेश किया गया था। यह आपको एक भौतिक सर्वर के सीपीयू (CPU) पर कई वर्चुअल मशीन (VM) चलाने की अनुमति देता है। वर्चुअलाइजेशन एप्लिकेशनों  को वीएम (VM) के बीच अलगाव उपलब्ध करता है और सुरक्षा का एक स्तर प्राप्त कराता है, क्योंकि एक एप्लिकेशन की जानकारी को दूसरे एप्लिकेशन द्वारा स्वतंत्र रूप से प्राप्त नहीं किया जा सकता है।
 
-वर्चुअलाइजेशन एक भौतिक सर्वर में संसाधनों के बेहतर उपयोग की अनुमति देता है और बेहतर मापनीयता की अनुमति देता है क्योंकि एक एप्लिकेशन को आसानी से जोड़ा या अपडेट किया जा सकता है, हार्डवेयर लागत को कम करता है, और बहुत कुछ। वर्चुअलाइजेशन के साथ आप भौतिक संसाधनों का एक सेट डिस्पोजेबल वर्चुअल मशीनों के समूह के रूप में प्रस्तुत कर सकते हैं।
+वर्चुअलाइजेशन एक भौतिक सर्वर में संसाधनों के बेहतर उपयोजन की अनुमति देता है और बेहतर मापक्रमणीयता की अनुमति देता है क्योंकि एक एप्लिकेशन को आसानी से जोड़ा या अपडेट किया जा सकता है, हार्डवेयर की लागत को कम करता है, और बहुत कुछ। वर्चुअलाइजेशन के साथ आप भौतिक संसाधनों का एक सेट प्रयोगशील वर्चुअल मशीनों के समूह के रूप में प्रस्तुत कर सकते हैं।
```
Comment (kundan2707) [inline content/hi/docs/concepts/overview/_index.md:35]: Existing one seems more correct no need of change

File: content/hi/docs/concepts/overview/_index.md [hunk 5]
```diff
@@ -21,70 +21,70 @@ sitemap:
...
+प्रत्येक वर्चुअल मशीन एक पूरी मशीन की तरह होती है, जिसमें अपने ऑपरेटिंग सिस्टम के साथ सभी घटक भी होते हैं, और ये सभी वर्चुअलाइज्ड हार्डवेयर पर चलते हैं।
 
-**कंटेनर डिप्लॉयमेंट युग:** कंटेनर VMs के समान होते हैं, लेकिन उनके पास एप्लिकेशनो के बीच ऑपरेटिंग सिस्टम (OS) को साझा करने के लिए अलगाव गुण होते हैं। इसलिए, कंटेनरों को हल्का माना जाता है। वीएम (VM) के समान, एक कंटेनर का अपना फाइल सिस्टम, सीपीयू का हिस्सा, मेमोरी, प्रोसेस स्पेस और बहुत कुछ होता है। चूंकि वे अंतर्निहित बुनियादी ढांचे से अलग हो गए हैं, वे बादलों और ओएस (OS) वितरण में पोर्टेबल हैं।
+**कंटेनर डिप्लॉयमेंट दौर:** कंटेनर VMs के समान होते हैं, लेकिन उनके पास एप्लिकेशनों के बीच ऑपरेटिंग सिस्टम को साझा करने के लिए एकाकीपन की स्वतंत्रता होती है। इसलिए, कंटेनरों को हल्का माना जाता है। वीएम (VM) के समान, एक कंटेनर का अपना फाइल सिस्टम, सीपीयू का हिस्सा, मेमोरी, प्रोसेस स्पेस और बहुत कुछ होता है। क्योंकि वे अंतर्निहित अवसंरचनाओं से अलग हो गए हैं, वे क्लाउड और ऑपरेटिंग सिस्टम वितरण में पोर्टेबल हैं।
 
-कंटेनर लोकप्रिय हो गए हैं क्योंकि वे अतिरिक्त लाभ प्रदान करते हैं, जैसे:
+कंटेनर लोकप्रिय हो गए हैं क्योंकि वे कई लाभ प्रदान करते हैं, जैसे:
```
Comment (kundan2707) [inline content/hi/docs/concepts/overview/_index.md:41]: No change required

File: content/hi/docs/concepts/overview/_index.md [hunk 6]
```diff
@@ -21,75 +21,75 @@ sitemap:
...
+* समर्थित एप्लिकेशनों के प्रकारों को सीमित नहीं करता है। कुबेरनेट्स का उद्देश्य स्टेटलेस, स्टेटफुल और डेटा-प्रोसेसिंग वर्कलोड सहित अत्यंत विविध प्रकार के वर्कलोड का समर्थन करना है। यदि कोई एप्लिकेशन कंटेनर में चल सकता है, तो उसे कुबेरनेट्स पर बहुत अच्छा चलना चाहिए।
+* सोर्स कोड डेप्लॉय नहीं करता है और आपके एप्लिकेशन का निर्माण नहीं करता है। कंटीन्यूअस इंटीग्रेशन, डिलीवरी और डिप्लॉयमेंट(CI/CD) कार्यप्रवाह संगठन संस्कृतियों और प्राथमिकताओं के साथ-साथ तकनीकी आवश्यकताओं द्वारा निर्धारित किए जाते हैं।
+* एप्लिकेशन-स्तरीय सेवाएं प्रदान नहीं करता है, जैसे कि मिडलवेयर (जैसे message buses), डेटा-प्रोसेसिंग फ्रेमवर्क (जैसे spark), डेटाबेस (जैसे MySQL), कैश मेमोरी, और न ही क्लस्टर स्टोरेज सिस्टम (जैसे की Ceph) अन्तर्निर्मित सेवाओं के रूप में कुबेरनेट्स पर चल सकते हैं। कुबेरनेट्स पर चल रहे एप्लिकेशनों को [ओपन सर्विस ब्रोकर](https://openservicebrokerapi.org/) जैसे पोर्टेबल तंत्रों के माध्यम से एक्सेस किया जा सकता है।
```
Comment (kundan2707) [inline content/hi/docs/concepts/overview/_index.md:83]: No change required

File: content/hi/docs/concepts/overview/_index.md [hunk 7]
```diff
@@ -21,75 +21,75 @@ sitemap:
...
 
 ## कुबेरनेट्स आपको प्रदान करता है:
 
 * **सेवा की खोज और लोड संतुलन**
-कुबेरनेट्स DNS नाम का उपयोग करके या अपने स्वयं के आईपी (IP) पते का उपयोग करके एक कंटेनर को उजागर कर सकता हैं। यदि एक कंटेनर में ट्रैफ़िक अधिक है, तो कुबेरनेट्स लोड बैलेंस करने और नेटवर्क ट्रैफ़िक को वितरित करने में सक्षम है ताकि डिप्लॉयमेंट स्थिर हो।
+कुबेरनेट्स DNS के नाम का उपयोग करके या अपने स्वयं के आईपी (IP) एड्रेस का उपयोग करके एक कंटेनर को प्रकट कर सकता है। यदि एक कंटेनर में ट्रैफ़िक अधिक है, तो कुबेरनेट्स लोड बैलेंस करने और नेटवर्क ट्रैफ़िक को वितरित करने में सक्षम है ताकि डिप्लॉयमेंट स्थिर हो।
 * **स्टोरेज ऑर्केस्ट्रेशन**
-कुबेरनेट्स आपको अपनी पसंद के स्टोरेज सिस्टम को स्वचालित रूप से माउंट करने की अनुमति देता है, जैसे कि स्थानीय स्टोरेज, पब्लिक क्लाउड प्रोवाइडर, और बहुत कुछ।
+कुबेरनेट्स आपको अपनी पसंदीदा स्टोरेज सिस्टम को स्वचालित रूप से माउंट करने की अनुमति देता है, जैसे कि स्थानीय स्टोरेज, पब्लिक क्लाउड प्रोवाइडर, और बहुत कुछ।
```
Comment (kundan2707) [inline content/hi/docs/concepts/overview/_index.md:65]: existing one is equally correct

File: content/hi/docs/concepts/overview/_index.md [hunk 8]
```diff
@@ -21,75 +21,75 @@ sitemap:
...
+* निरंतर विकास, एकीकरण और डिप्लॉयमेंट: विश्वसनीय और नियमित कंटेनर इमेज निर्माण और त्वरित और कुशल रोलबैक का समर्थन (इमेज अपरिवर्तनीयता के कारण) प्रदान करता है।
+* Dev और Ops चिंताओं का पृथकीकरण: डिप्लॉयमेंट समय के बजाय बिल्ड/रिलीज़ समय पर एप्लिकेशन कंटेनर इमेजेस (images) बनाएं, जिससे अवसंरचना से एप्लिकेशनों को अलग किया जा सके।
+* अवलोकन:यह न केवल ओएस-स्तर की जानकारी और मेट्रिक्स को सतह पर रखता है, बल्कि एप्लिकेशन के स्वास्थ्य और अन्य संकेतों को भी लागू करता है।
+* विकास, परीक्षण और उत्पादन में पर्यावरणीय संगतता: लैपटॉप पर वैसे ही चलता है जैसे क्लाउड में चलता है।
+* क्लाउड और ओएस (OS) वितरण पोर्टेबिलिटी: Ubuntu, RHEL, CoreOS, On-Premises, प्रमुख सार्वजनिक क्लाउड पर और अन्य कहीं भी चलता है।
+* एप्लिकेशन-केंद्रित प्रबंधन: वर्चुअल हार्डवेयर पर OS चलाने से लेकर तार्किक संसाधनों का उपयोग करके OS पर एप्लिकेशन चलाने तक अमूर्तता के स्तर को बढ़ाता है।
+* Loosely coupled, Distributed, Elastic-free, Microservices: एप्लिकेशन को छोटे, स्वतंत्र टुकड़ों में तोड़ा जाता है और उन्हें सक्रिय रूप से तैनात और प्रबंधित किया जा सकता है - एक बड़ी एकल-उद्देश्य वाली मशीन पर चलने वाला एक मोनोलिथिक स्टैक नहीं है।
```
Comment (divya-mohan0209) [inline content/hi/docs/concepts/overview/_index.md:50]: I would prefer the transliterated versions for the last line as well as the previous lines.

File: content/hi/docs/concepts/overview/_index.md [hunk 9]
```diff
@@ -21,75 +21,75 @@ sitemap:
...
+कुबेरनेट्स आपको अपनी पसंदीदा स्टोरेज सिस्टम को स्वचालित रूप से माउंट करने की अनुमति देता है, जैसे कि स्थानीय स्टोरेज, पब्लिक क्लाउड प्रोवाइडर, और बहुत कुछ।
 * **स्वचालित रोलआउट और रोलबैक**
 आप कुबेरनेट्स का उपयोग करके अपने तैनात कंटेनरों के लिए वांछित स्थिति का वर्णन कर सकते हैं, और यह वास्तविक स्थिति को नियंत्रित दर पर वांछित स्थिति में बदल सकता है। उदाहरण के लिए, आप अपने डिप्लॉयमेंट के लिए नए कंटेनर बनाने के लिए कुबेरनेट्स को स्वचालित कर सकते हैं, मौजूदा कंटेनरों को हटा सकते हैं और उनके सभी संसाधनों को नए कंटेनर में अपना सकते हैं।
 * **स्वचालित बिन पैकिंग **
-आप कुबेरनेट्स को नोड्स के एक समूह के साथ प्रदान करते हैं जिसका उपयोग वह कंटेनरीकृत कार्यों को चलाने के लिए कर सकता है। आप कुबेरनेट्स को बताते हैं कि प्रत्येक कंटेनर को कितना सीपीयू और मेमोरी (रैम) चाहिए। कुबेरनेट्स आपके संसाधनों का सर्वोत्तम उपयोग करने के लिए कंटेनरों को आपके नोड्स में फिट कर सकता है।
+आप कुबेरनेट्स को नोड्स के एक समूह के साथ प्रदान करते हैं जिसका उपयोग वह कंटेनरीकृत कार्यों को चलाने के लिए कर सकता है। आप कुबेरनेट्स को बताते हैं कि प्रत्येक कंटेनर को कितना सीपीयू और मेमोरी (रैम) चाहिए। आपके संसाधनों का सर्वोत्तम उपयोग करने के लिए कंटेनरों को आपके नोड्स में फिट कर सकता है।
```
Comment (divya-mohan0209) [inline content/hi/docs/concepts/overview/_index.md:69]: Why is this change required?

File: content/hi/docs/concepts/overview/_index.md [hunk 10]
```diff
@@ -21,75 +21,75 @@ sitemap:
...
+इस तरह कुबेरनेट्स बचाव के लिए आता है! कुबेरनेट्स आपको वितरित सिस्टम को लचीलेपन से चलाने के लिए एक ढांचा प्रदान करता है। यह आपके एप्लिकेशन के स्केलिंग और फेलओवर का ख्याल रखता है, डिप्लॉयमेंट पैटर्न के साथ और भी बहुत कुछ प्रदान करता है। उदाहरण के लिए, कुबेरनेट्स आपके सिस्टम के लिए कैनरी डिप्लॉयमेंट को आसानी से प्रबंधित कर सकता है।
 
 ## कुबेरनेट्स आपको प्रदान करता है:
 
 * **सेवा की खोज और लोड संतुलन**
-कुबेरनेट्स DNS नाम का उपयोग करके या अपने स्वयं के आईपी (IP) पते का उपयोग करके एक कंटेनर को उजागर कर सकता हैं। यदि एक कंटेनर में ट्रैफ़िक अधिक है, तो कुबेरनेट्स लोड बैलेंस करने और नेटवर्क ट्रैफ़िक को वितरित करने में सक्षम है ताकि डिप्लॉयमेंट स्थिर हो।
+कुबेरनेट्स DNS के नाम का उपयोग करके या अपने स्वयं के आईपी (IP) एड्रेस का उपयोग करके एक कंटेनर को प्रकट कर सकता है। यदि एक कंटेनर में ट्रैफ़िक अधिक है, तो कुबेरनेट्स लोड बैलेंस करने और नेटवर्क ट्रैफ़िक को वितरित करने में सक्षम है ताकि डिप्लॉयमेंट स्थिर हो।
```
Comment (divya-mohan0209) [inline content/hi/docs/concepts/overview/_index.md:63]: I don't think address needs to be transliterated. Rest of the changes, I agree with.
```suggestion
कुबेरनेट्स DNS के नाम का उपयोग करके या अपने स्वयं के आईपी (IP) पते का उपयोग करके एक कंटेनर को प्रकट कर सकता है। यदि एक कंटेनर में ट्रैफ़िक अधिक है, तो कुबेरनेट्स लोड बैलेंस करने और नेटवर्क ट्रैफ़िक को वितरित करने में सक्षम है ताकि डिप्लॉयमेंट स्थिर हो।
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
Initial set of reviews

**Comment by kundan2707** [comment]
@AnshRoshan gentle reminder

**Pull request closed without merging**